tests/lib/rules/no-tabs.js

```diff
@@ -23,11 +23,46 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n}",
         "function test(){\n" +
         "  //   sdfdsf \n" +
-        "}",
+        "}"
     ],
     invalid: [
+
+        // Tabs in string/template/regex literals should be reported but not auto-fixed
```

Comment:
I suppose this isn't a big issue, but is there a reason why we would explicitly not replace tabs in literals, rather than replacing the tabs with '\t'?